packages/icons/src/Icon.js

```diff
@@ -10,29 +10,33 @@ const Icon = ({ name, title, desc, titleId, descId, ...props }) => {
   ariaLabelledBy += desc && descId ? ` ${descId}` : ''
   ariaLabelledBy = ariaLabelledBy ? ariaLabelledBy : undefined
 
-  return (
-    Component && (
+  if (!Component) {
+    if (process.env.NODE_ENV !== 'production')
+      console.trace(`icon ${iconName} does not exist`)
+    return null
+  } else {
+    return (
       <Component
+        aria-hidden="true"
         title={title}
         desc={desc}
         titleId={titleId}
         descId={descId}
-        aria-hidden={!!ariaLabelledBy}
+        aria-hidden={!!ariaLabelledBy || true}
```

Comment:
looks like aria-hidden is specified twice in this Component

Reply:
@jes708, only once... I've deleted the aria-hidden from default props, and use it as a default case here.

The thought behind it is to use less default props with weird object key quoting like 'aria-hidden', and use the aria-hidden = true as a fault back value by `||` operator, when aria labelled by value is not valid.